src/main/java/com/salesforce/dataloader/ui/UIUtils.java

```diff
@@ -26,27 +26,39 @@
 
 package com.salesforce.dataloader.ui;
 
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.util.Arrays;
-import java.util.List;
+import com.salesforce.dataloader.action.OperationInfo;
 
 import org.eclipse.jface.resource.ImageDescriptor;
 import org.eclipse.jface.resource.ImageRegistry;
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.graphics.Color;
-import org.eclipse.swt.widgets.*;
+import org.eclipse.swt.widgets.Combo;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.MessageBox;
+import org.eclipse.swt.widgets.Shell;
 
-import com.salesforce.dataloader.action.OperationInfo;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
 
 public class UIUtils {
     private static ImageRegistry image_registry;
 
-    public static URL newURL(String url_name) {
+    public static boolean isValidHttpsUrl(String url) {
         try {
-            return new URL(url_name);
-        } catch (MalformedURLException e) {
-            throw new RuntimeException("Malformed URL " + url_name, e);
+            // check if it is a valid url
+            new URL(url).toURI();
+            // check if it is https protocol
+            return url.matches("^https://.*$");
```

Comment:
This will not match casing.
It would be better to use this [Uri](https://docs.oracle.com/javase/7/docs/api/java/net/URI.html#getScheme())

Reply:
Sounds better.